cmd/cluster/aws/create.go

```diff
@@ -55,6 +54,21 @@ func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
 			defer cancel()
 		}
 
+		if len(opts.CredentialSecretName) == 0 {
+			if err := isRequiredOption("aws-creds", opts.AWSPlatform.AWSCredentialsFile); err != nil {
+				return err
+			}
+			if err := isRequiredOption("pull-secret", opts.PullSecretFile); err != nil {
+				return err
+			}
+		} else {
+			//Check the secret exists now, otherwise stop
```

Comment:
nit: comments by convention leave space after and end with a period.
e.g `// Check the secret exists now, otherwise stop.`